src/daos/docker/ceph/redhat.repo

```diff
@@ -0,0 +1,96 @@
+[EOS_Puppet6_puppet6-el8]
```

Comment:
Does this need to be in the repo? Is it not on the ssc machines?

Reply:
I basically took the repo file from one of the machines that had it and changed the https requests to http (with Tim's help).
I didn't look at the contents to pick and choose what to include, I just included everything from that Seagate repo.